filebeat/harvester/log.go

```diff
@@ -99,6 +99,13 @@ func (h *Harvester) Harvest() {
 			event.Text = &text
 			event.JSONFields = message.Fields
 			event.EventMetadata = h.config.EventMetadata
+			if message.Fields != nil {
```

Comment:
Fixed. @urso what do you mean be no need to run Fields.Update? Is this for the code below?